### PR TITLE
Add requires.io

### PR DESCRIPTION
**Pull request by pbteja1998**
# What kind of change does this PR introduce
This PR adds `requires.io` to the repository to track dependencies and stay up-to-date with all the dependencies.

## What is the current behavior
- There is no package in the repo to track dependencies.
- Right now, we are adding all the requirements and dependencies of those requirements to `requirements.txt`

## What is the new behavior
- `Requires.io ` automatically sends a pull request if a dependency of the project becomes out of date.
- There is also a badge which tells whether the requirements of the repository are up-to-date or not.
- From now, only the main packages will be added to the `requirements.txt` file. Corresponding guidelines in `CONTRIBUTING.md` file is changed accordingly.

### Checklist for this PR

* [x] I have read [contributing guidelines](https://github.com/coderplex/coderplex-backend/blob/develop/.github/CONTRIBUTING.md).
* [x] Updated the requirements (`pip freeze > requirements/requirements.txt`). 
* [N/A] Created new migrations (`python webapp/manage.py makemigrations && python webapp/manage.py migrate`).
* [x] Pull request is towards `develop` branch.
* [x] Documentation
* [x] Ready to be merged.

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
you guys are sending  pr at awesome rate . ;-)

**Comment by pbteja1998** [comment]
@rrmerugu  Do you know any other better package than `requires.io` that does this task?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
+[![flake8 code style](https://img.shields.io/badge/code_style-flake8-5ed9c7.svg)](http://flake8.pycqa.org/en/latest/) [![formatted with yapf](https://img.shields.io/badge/code_formatted_with-yapf-ff69b4.svg)](https://github.com/google/yapf) [![Build Status](https://travis-ci.org/coderplex/coderplex-backend.svg?branch=develop)](https://travis-ci.org/coderplex/coderplex-backend) [![Requirements Status](https://requires.io/github/coderplex/coderplex-backend/requirements.svg?branch=develop)](https://requires.io/github/coderplex/coderplex-backend/requirements/?branch=develop)
+ [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://github.com/coderplex/coderplex/blob/master/LICENSE)
```
Comment (vinaypuppal) [inline README.md]: @pbteja1998 Can you also fix this small typo `https://github.com/coderplex/coderplex/blob/master/LICENSE` to `https://github.com/coderplex/coderplex-backend/blob/master/LICENSE` Which i missed in #22

Comment (pbteja1998) [reply README.md]: Done

**Review by vinaypuppal — APPROVED** [review]

**Pull request merged**